src/entities/campaign/components/CampaignBanner.tsx

```diff
@@ -113,7 +115,11 @@ export const CampaignBanner: React.FC<CampaignBannerProps> = ({ campaignId }) =>
             }
             {...{ campaignId }}
           />
-          <SocialsShare variant="button" />
+          <SocialsShare
+            shareText={`Support ${campaign?.name} Campaign on ${PLATFORM_NAME}  by donating or 
```

Comment:
```suggestion
            shareText={`Support ${campaign?.name} Campaign on ${PLATFORM_NAME} by donating or 
```
nitpick: Redundant whitespace

Reply:
resolved